### ISSUE TITLE
Bluetooth address resolution and P2P connection

### ISSUE THREAD

**Issue by varun-dhar** [airalex/Servus#1]
Hi, I'm trying to do zeroconf communication over bluetooth between a windows/linux pc and an ios device. I have a few questions about using this library for that purpose.
When Servus discovers a device over bluetooth, what kind of hostname is returned?

After a bluetooth hostname is returned, can I use it with HTTP as outlined in the README?

Do I advertise with bonjour/avahi on my other device?

In short, how do I connect the devices and have them communicate?

Thanks in advance

By the way, the library seems pretty cool! Its hard to find good resources/libraries (like this one) for this sort of stuff.

**Comment by mike-dube** [comment]
Hi @doggo4242, did you have any progress on that matter?

We are looking to reach a HTTP server, hosted on Windows from an iOS device, but they cannot be on the same WiFi.

**Comment by varun-dhar** [comment]
i gave up 2 weeks after i posted this issue lol
however, i may have some insights. it seems that you cant do much without a bt connection; without it, youre limited to sending 16 bytes at a time encoded in the manufacturer data of a ble beacon, which is quite slow. with one, you can send data much faster, more than likely fast enough for basic web browsing. i was looking into l2cap before i got busy and gave up.
hope that helps.
